### DISCUSSION TITLE
PyLoad DDownload Cert Error

### DISCUSSION THREAD

**Discussion by itsNik** [pyload/pyload#4367] [Help] [answered]
Related to this issue:
https://github.com/pyload/pyload/issues/4273

In the newest container version (python 3.11) there is no cacert.pem file to implement this workaround.

Does anyone know how to fix this issue?

**Comment by defencedog** [comment]
I am still having the same issue. Captcha loads, file size is properly retrieved but download don't start

reference URL
> https://ddownload.com/dyqe3kz4cz0v/Indurial_Inrumentation.part1.rar

**Reply by GammaC0de** [reply]
Please send debug log

**Reply by defencedog** [reply]
[debug_DdownloadCom_2025-11-21_22-42-01.zip](https://github.com/user-attachments/files/23683253/debug_DdownloadCom_2025-11-21_22-42-01.zip)

@GammaC0de

**Comment by GammaC0de** [comment]
This is duplicate of https://github.com/pyload/pyload/issues/4469

**Comment by defencedog** [comment]
as per this solution https://github.com/pyload/pyload/issues/4273#issuecomment-1539164273

I used firefox to download certificates `.pem` from ddownload.com which are attached
Then updated `/home/ukhan/.local/lib/python3.10/site-packages/certifi/cacert.pem` keeping in view that this is where `pyload` is installed
Restart pyload but **no success**

[ddownload-com.zip](https://github.com/user-attachments/files/23685932/ddownload-com.zip)

**Comment by defencedog** [comment]
**Alternative** as per this solution https://github.com/pyload/pyload/issues/4273#issuecomment-1581939788
Changed extension of above downloaded `.pem` files to `.crt` files
moved files to `/usr/local/share/ca-certificates/` & ran `sudo update-ca-certificates`
Got confirmation 
```
2 added, 0 removed; done.
Running hooks in /etc/ca-certificates/update.d...
/etc/ssl/certs/adoptium/cacerts successfully populated.

Adding debian:ddownload-com2.pem
Adding debian:ddownload-com.pem
done.
```
Restarted `pyload` but **no success**

**Comment by GammaC0de** [comment, marked as answer]
https://github.com/pyload/pyload/commit/e93397788a4006214c81b9b83b8a19ba34998dd7
You have to set "Verify SSL certificates" config option to "on (using aia-chaser)".
